templates/car.html

```diff
@@ -1,40 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body style="background-color: #e5e5e5;">
+    <div>
+        <h1 class="header">Add Car Information</h1>
+    
+        <label for="vin">VIN</label>
+        <input id="vin" type="text" />
+        <br>
+        <label for="cartype">Car Type</label>
+        <select id="cartype">
+            <option value="SU">SUV</option>
+            <option value="SE">Sedan</option>
+            <option value="CO">Coupe</option>
+            <option value="MV">Mini Van</option>
+            
+        </select>
+        <br>
+        <label for="make">Make</label>
+        <select id="make">
+            <option value="Audi">Audi</option>
+            <option value="BMW">BMW</option>
+            <option value="Citroen">Citroen</option>
+            <option value="Ford">Ford</option>
+            <option value="Honda">Honda</option>
+            <option value="Jaguar">Jaguar</option>
+            <option value="Land Rover">Land Rover</option>
+            <option value="Mercedes">Mercedes</option>
+            <option value="Mini">Mini</option>
+            <option value="Nissan">Nissan</option>
+            <option value="Toyota">Toyota</option>
+            <option value="Volvo">Volvo</option>
+          </select>
+        <br>
+        <label for="model">Model</label>
+        <input id="model" type="text" />
+        <br>
+        <label for="year">Year</label>
+        <input id="year" type="text" />
```

Comment:
nit: since we're styling/updating the page. Make the type be number
```suggestion
        <input id="year" type="number" />
```